content/projects/java-specific/threads/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+content_type: project
+flavours:
+- java
+prerequisites:
+  hard:
+  - topics/java-specific/concurrency
+  - topics/java-specific/functional-interface
+  soft:
+  - projects/java-specific/functional-interface
+ready: true
+submission_type: repo
+title: Java Threads
+---
+
+### The Chicken and the Farmer
+
+Have you read the book Animal Farm well its not so important for this project but if you havent read it you should; anyways, in this project you are going to create a Chicken and a Farmer each of these will be there own functions and these are the rules
```

Comment:
This is missing a bit of punctuation. 

Here's a revised version. I also tried to interpret what you meant, I might be wrong. But if I was wrong then the learners are definitely going to get it wrong.

```
Have you read the book Animal Farm? Well, it's not so important for this project, but if you haven't read it you should.  Anyways, in this project, you are going to create two classes,  `Chicken` and  `Farmer`.  They should function as follows:
```